slo_lambda/main.tf

```diff
@@ -120,3 +124,32 @@ resource "aws_lambda_permission" "allow_cloudwatch_to_call_windowed_slo" {
   principal     = "events.amazonaws.com"
   source_arn    = aws_cloudwatch_event_rule.every_one_day.arn
 }
+
+resource "aws_cloudwatch_dashboard" "sli" {
+  dashboard_name = "${var.env_name}-sli"
+  dashboard_body = jsonencode({
+    "widgets" : [for k, v in var.slis :
+      {
+        "type" : "metric"
+        "width" : 24
+        "height" : 9
+        "properties" : {
+          "view" : "timeSeries"
+          "stacked" : false
+          "metrics" : [
+            [local.namespace, "${var.sli_prefix}-${k}"]
+          ]
+          "region" : data.aws_region.current.name
+          "title" : "${v.description != null ? v.description : k} over last ${var.window_days} days"
+          "stat": "Average"
+          "period" : 24 * 60 * 60
+          "yAxis": {
+            "left": {
+              "showUnits": false
+            }
```

Comment:
```suggestion
            "left": {
              "label": "% Passing"
              "showUnits": false
            }
```

Reply:
The "No Unit" label is annoying to me.  Not sure what to call this...  Is a more accurate label "% Passing in last 28 days"?

Reply:
Sorry, the screenshot is out of date. There's no currently no unit label at all, since ratios don't technically have units.